src/commands/vote.ts

```diff
@@ -14,6 +14,6 @@ export default class extends Command {
   }
 
   async exec(message: FireMessage) {
-    await message.channel.send(constants.url.FireVote);
+    return await message.channel.send(constants.url.FireVote);
```

Comment:
```suggestion
    return await message.channel.send(constants.url.FireVote);
```
adding return changed nothing about the function of the command

Reply:
sorry, guess I shouldnt have tried to help since im loosing brain cells right now and cant think

Reply:
ill close the pull request